### ISSUE TITLE
Complete run - epoch 100 - model F-M - Only auto adjust levels colors

### ISSUE THREAD

**Issue by MaraniMatias** [deeplearningrosario/deeplearningforcomputervision#41]
# Runs
**Image layer:** ResNet50
**Image quantity:** 
- Males: 6780 (4520 training - 1130 validation - 1130 testing)
- Females: 5740 (3828 training - 956 validation - 956 testing)

**Epochs**: 100
**Batch size:** 32
**Loss**: Mean Squared Error
**Additional metrics:** Mean Absolute Error
**Optimizer:** Adam(lr=0.001, amsgrad=True)
**Dropout:** 0.45

## Female && male
**Test loss:** 165.81839496908802
**Test MAE:** 9.841582104435131

![history_loss](https://user-images.githubusercontent.com/7859628/45230985-fe8d8880-b2a0-11e8-971f-a0454212b2be.png)
![history_mean](https://user-images.githubusercontent.com/7859628/45230986-ffbeb580-b2a0-11e8-802f-fd979bd467f0.png)

epoch | loss | lr | mean_absolute_error | val_loss | val_mean_absolute_error
-- | -- | -- | -- | -- | --
0 | 2318.38091804128 | 0.001 | 35.9602790717366 | 1778.90623782808 | 35.5175752429468
1 | 670.637376522984 | 0.001 | 19.7752686017543 | 3329.98803547085 | 47.8861924497873
2 | 437.534811432075 | 0.001 | 16.0848058445578 | 2631.01672445208 | 45.3361513534648
3 | 375.514644591729 | 0.001 | 14.9444279069814 | 332.769502400209 | 14.3371405395771
4 | 373.28169276078 | 0.001 | 15.0295553892837 | 1514.40894900621 | 29.0112041645288
5 | 332.102591601867 | 0.001 | 14.1892128054697 | 1087.01714684111 | 23.7400184697181
6 | 323.717291049199 | 0.001 | 13.9285700573668 | 1124.79282034621 | 27.4761751637591
7 | 242.341707340198 | 0.00080000004 | 12.1853301875832 | 323.662428595304 | 13.5689822916559
8 | 247.098484801515 | 0.00080000004 | 12.3040695496592 | 229.269118926212 | 11.6022721132405
9 | 220.80236674566 | 0.00080000004 | 11.6139823377104 | 642.708004506185 | 22.1144125052404
10 | 236.481412418733 | 0.00080000004 | 12.039224216569 | 658.798259321818 | 20.7075231539323
11 | 220.741777303976 | 0.00080000004 | 11.6348437879094 | 634.041951399864 | 20.9861484117956
12 | 193.196879826954 | 0.00064000004 | 10.8674255644355 | 213.751931794881 | 11.0562792144366
13 | 188.273321081362 | 0.00064000004 | 10.7402046922867 | 177.05358567791 | 10.0154643739973
14 | 187.604829522258 | 0.00064000004 | 10.743011107214 | 412.543583110103 | 15.4342180948815
15 | 187.750114159431 | 0.00064000004 | 10.7449430027931 | 617.155247396614 | 19.9028707359804
16 | 170.010506250211 | 0.00064000004 | 10.2287027494269 | 166.457188870511 | 9.87050097969326
17 | 161.859411476329 | 0.00064000004 | 9.99513466619794 | 154.584557976727 | 9.29133608242924
18 | 145.573290862818 | 0.00064000004 | 9.44713857520067 | 217.373366241601 | 11.3200318289855
19 | 154.090488954772 | 0.00064000004 | 9.7009499389473 | 162.01639010465 | 9.51247804665497
20 | 136.811066386685 | 0.00064000004 | 9.1303382717255 | 295.594035352522 | 13.2603690631095
21 | 126.832499044845 | 0.00051200006 | 8.82217426144683 | 181.902369351995 | 10.2331218298878
22 | 116.815034223848 | 0.00051200006 | 8.47086121879472 | 209.633754887494 | 11.1878317424229
23 | 117.234350382682 | 0.00051200006 | 8.39557511965282 | 172.643503015359 | 9.95449513099795
24 | 106.050706085658 | 0.00040960003 | 8.00450763373382 | 152.097541523803 | 9.26361809343757
25 | 122.596980505716 | 0.00040960003 | 8.54792847836709 | 145.478303435665 | 9.00915982602084
26 | 107.250429051997 | 0.00040960003 | 8.02750749272711 | 160.774437440779 | 9.58810014807024
27 | 97.6631861468737 | 0.00040960003 | 7.61702417447685 | 191.825831470874 | 10.3507035567228
28 | 100.462029764626 | 0.00040960003 | 7.72433198280865 | 152.377812210917 | 9.27275369905671
29 | 91.8309795597151 | 0.00032768003 | 7.4190299537231 | 155.617149140897 | 9.26252222198311
30 | 84.2584297589611 | 0.00032768003 | 7.11960472395732 | 163.991698867637 | 9.53256580356441
31 | 82.6873703405054 | 0.00032768003 | 6.98093968872597 | 169.938365903362 | 9.68294011552176
32 | 79.7434501044763 | 0.00026214402 | 6.84308260299398 | 154.267452558186 | 9.29673566662797
33 | 77.4900613701623 | 0.00026214402 | 6.77239135171902 | 150.275134312455 | 9.24379886122472
34 | 72.7022689852237 | 0.00026214402 | 6.53608312410319 | 151.677254944725 | 9.31985849250654
35 | 69.534037149061 | 0.00020971522 | 6.41543681688953 | 154.476166312784 | 9.27660382399737
36 | 69.7854037483628 | 0.00020971522 | 6.36961143714821 | 166.81994334849 | 9.61367089407785
37 | 65.9898020657958 | 0.00020971522 | 6.20611102442794 | 157.312489832351 | 9.41051653681871
38 | 66.0959788616274 | 0.00016777217 | 6.21175447170621 | 149.201684836573 | 9.19342713982353
39 | 63.8599998609381 | 0.00016777217 | 6.08416644636958 | 153.200523208223 | 9.18853508134588
40 | 61.661143139783 | 0.00016777217 | 6.00491176796234 | 161.629126441787 | 9.48403020055959
41 | 59.46432794517 | 0.00013421774 | 5.86922763258747 | 151.54212606399 | 9.22334156877585
42 | 58.6770152022065 | 0.00013421774 | 5.80145917101002 | 155.997776722748 | 9.3895798095235
43 | 56.2653465508615 | 0.00013421774 | 5.74416969995723 | 159.802506385576 | 9.46322800443362
44 | 55.9813080126438 | 0.000107374195 | 5.66584576061598 | 156.905971157699 | 9.39974497925858
45 | 54.6244830253539 | 0.000107374195 | 5.66031648190786 | 158.036067212874 | 9.36758599413893
46 | 57.2123400921678 | 0.000107374195 | 5.75518773798173 | 164.173157440507 | 9.57909071731202
47 | 54.3707208747663 | 0.0001 | 5.58911402464256 | 158.383871778889 | 9.35661513419074
48 | 56.3729232510255 | 0.0001 | 5.6940705439207 | 164.737366146918 | 9.66776503218215
49 | 53.264162870589 | 0.0001 | 5.51468974251923 | 159.36726304783 | 9.44556378358162
50 | 53.4858683525345 | 0.0001 | 5.55127067227768 | 166.900497948563 | 9.66191418173215
51 | 53.0828416419475 | 0.0001 | 5.53496299112134 | 181.340503974249 | 10.1118447110843
52 | 52.446886266426 | 0.0001 | 5.53048549820261 | 157.162841445762 | 9.41943683788852
53 | 50.8866563805222 | 0.0001 | 5.41530481479245 | 160.765901574695 | 9.54889897029215
54 | 49.5816765251946 | 0.0001 | 5.37470336182014 | 161.530149816438 | 9.5457408069307
55 | 51.5025119872886 | 0.0001 | 5.47739142096665 | 165.268815110994 | 9.71376705352549
56 | 49.733044654031 | 0.0001 | 5.39837148055884 | 157.321547876726 | 9.42346480068744
57 | 51.1331228650125 | 0.0001 | 5.47309578487504 | 167.233497729352 | 9.73350679977285
58 | 50.3066775729111 | 0.0001 | 5.38847225593161 | 158.387680346305 | 9.41823267090926
59 | 48.9963286161766 | 0.0001 | 5.33508855370054 | 161.880441769886 | 9.61702642230494
60 | 47.9987981400538 | 0.0001 | 5.26181108527739 | 165.194710341076 | 9.62388659392199
61 | 46.3954733084079 | 0.0001 | 5.25043400018251 | 159.695055894861 | 9.52552458469614
62 | 48.6827220944166 | 0.0001 | 5.33740723572911 | 160.350366360161 | 9.56747147860943
63 | 45.4062207600804 | 0.0001 | 5.12786461076613 | 158.799117354396 | 9.51301169738331
64 | 48.9588187692493 | 0.0001 | 5.31238405340073 | 161.653897659433 | 9.57907183401194
65 | 47.829361570584 | 0.0001 | 5.28621989502882 | 156.796917879684 | 9.44111678744322
66 | 46.5383817200233 | 0.0001 | 5.22363240123652 | 159.089940481652 | 9.59208342258676
67 | 46.2891091774912 | 0.0001 | 5.1624418022419 | 160.466308959493 | 9.57049710501418
68 | 46.2904940734483 | 0.0001 | 5.16643445275877 | 162.662848716729 | 9.60342962424113
69 | 45.4518946769652 | 0.0001 | 5.14992570968467 | 159.968789192997 | 9.51267509332439
70 | 47.6994735999261 | 0.0001 | 5.24503457129259 | 168.248527461022 | 9.78307430947616
71 | 46.375828213365 | 0.0001 | 5.18158491319255 | 166.925898024357 | 9.77941329367209
72 | 44.7811426620593 | 0.0001 | 5.11443722642205 | 158.936546830409 | 9.49532726924234
73 | 45.6525521500007 | 0.0001 | 5.16800150393754 | 162.392911214271 | 9.64927505098162
74 | 44.7246242901556 | 0.0001 | 5.14638046813914 | 165.724882028933 | 9.70958011513817
75 | 44.814995563253 | 0.0001 | 5.11163201626831 | 160.879348945069 | 9.59159952019228
76 | 44.9254148800406 | 0.0001 | 5.16757304278869 | 156.415162186152 | 9.40977678728698
77 | 44.3496737569271 | 0.0001 | 5.08808901781778 | 158.955632064395 | 9.48713310384339
78 | 43.6582953339281 | 0.0001 | 5.05852362066579 | 159.236322638858 | 9.50947342302975
79 | 44.3194736376526 | 0.0001 | 5.09395341206008 | 163.769562820917 | 9.70798060763717
80 | 44.315785925755 | 0.0001 | 5.08287270216492 | 163.294082524564 | 9.61140088861306
81 | 43.7511679444178 | 0.0001 | 5.06259482524929 | 162.722822692227 | 9.62904604146488
82 | 44.6216332801632 | 0.0001 | 5.12070062849358 | 159.933076984702 | 9.51782526380149
83 | 44.4514541625977 | 0.0001 | 5.09850673990839 | 163.855738360953 | 9.62060010330332
84 | 42.4811355963699 | 0.0001 | 5.03205871582031 | 160.610695229052 | 9.54989808632109
85 | 44.2959175018471 | 0.0001 | 5.09858061082636 | 162.637390517091 | 9.62430022432614
86 | 43.46939884948 | 0.0001 | 5.03574792585935 | 160.245066861293 | 9.61359230326783
87 | 42.6753495458101 | 0.0001 | 5.01345355875137 | 157.942281560267 | 9.46862189127409
88 | 40.7484568955513 | 0.0001 | 4.91697680464645 | 160.973550725189 | 9.558398589649
89 | 42.8280615000046 | 0.0001 | 5.03039353786043 | 164.620249776575 | 9.71129273773954
90 | 43.1602448251868 | 0.0001 | 5.05874057311902 | 159.881776262907 | 9.57253871873835
91 | 41.0705645770963 | 0.0001 | 4.9256547572342 | 164.085741520385 | 9.71685463767267
92 | 40.6982727215287 | 0.0001 | 4.90030499423127 | 160.827762051488 | 9.55677266820394
93 | 40.6521637838425 | 0.0001 | 4.92159821659284 | 159.783384694349 | 9.54340921678113
94 | 41.605965218135 | 0.0001 | 4.94255194163631 | 160.785580394595 | 9.56040564067983
95 | 42.144915023716 | 0.0001 | 4.97286306001035 | 160.12101919699 | 9.60452233110613
96 | 41.5403733502522 | 0.0001 | 4.94332439315679 | 158.687355385302 | 9.56540666399158
97 | 41.4349230358232 | 0.0001 | 4.93361114575981 | 164.194667446762 | 9.76163212473532
98 | 40.461926303529 | 0.0001 | 4.86615367111659 | 159.807541390965 | 9.57191923457847
99 | 41.0888507278216 | 0.0001 | 4.88800481682939 | 167.330054110329 | 9.74851447440977
